packages/services/test/helpers/userNotificationTypeByModel.js

```diff
@@ -0,0 +1,5 @@
+import { UserNotificationType } from '@aragonone/court-backend-server/build/models/objection'
+
+export default function userNotificationTypeByModel(model) {
+  return UserNotificationType.query().findOne({model}).withGraphFetched('notifications')
```

Comment:
Would it be helpful to have it in the model? It seems like the fact of needing this for test is telling us something :)